test/aee.js

```diff
@@ -0,0 +1,47 @@
+require('../aee.js')
+const expect = require('chai').expect
+const sinon = require('sinon')
+const data = require('./testData')
+
+describe('Setting up the environment', () => {
+  it('checks if testing works', () => {
```

Comment:
The guys who wrote the `mocha` framework already did a good job testing it, please do not embarrass them.